backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/process/adprocess/ADProcessPostProcessService.java

```diff
@@ -378,15 +376,13 @@ else if (ViewOpenTarget.ModalOverlay.equals(target))
 				return OpenViewAction.builder()
 						.viewId(ViewId.ofViewIdString(viewToOpen.getViewId()))
 						.profileId(ViewProfileId.fromJson(viewToOpen.getProfileId()))
-						.modalOverlay(true)
```

Comment:
pls precisely set the targetTab, even if we have a default defined.
This will make the code more readable and robust.